hcp/preprocessing.py

```diff
@@ -62,6 +62,42 @@ def apply_ica_hcp(raw, ica_mat, exclude):
     raw._data /= 1e15
 
 
+def apply_ref_meg_residual_fit(raw, decim_fit=100):
+    """Regress out MEG ref channels
+
+    Computes linear models from MEG reference channels
+    on each sensors, predicts the MEG data and subtracts
+    and computes the residual by subtracting the predictions.
+    Note. Operates in place and can be memory demanding.
+    To alleviate this problem the model can be fit on decimated
+    data. This is permissive because the linear model does
+    not have any representation of time, only the distributions
+    matter.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The BTi/4D raw data.
+    decim_fit : int
+        The decimation factor used for fitting the model.
+        Defaults to 100.
+    """
+    from sklearn.linear_model import LinearRegression
```

Comment:
you would need to add `scikit-learn` as an optional dependency on the readme page